src/cli/bootupctl.rs

```diff
@@ -87,10 +90,7 @@ impl CtlCommand {
 
     /// Runner for `status` verb.
     fn run_status(opts: StatusOpts) -> Result<()> {
-        let mut client = ClientToDaemonConnection::new();
-        client.connect()?;
-
-        let r: Status = client.send(&bootupd::ClientRequest::Status)?;
+        let r = bootupd::status()?;
```

Comment:
It's definitely *nice* to be able to see status as an unprivileged user, but...let's try not to scope in functional changes a this time.  So my preference would be to change this call to also run the `ensure_running_in_systemd()` helper too for consistency.

Reply:
And then we can keep the test suite *unchanged* which helps reinforce the idea this is just an implementation change.

Reply:
Sure, revert the test suite about the normal user, and add user detecting in `fn require_root_permission()`, change the error msg to "error: Failed as no root permission" if none root.
WDYT?